### ISSUE TITLE
bug: arkade get inletsctl fails in linux aarch64 and GitBash

### ISSUE THREAD

**Issue by joebowbeer** [alexellis/arkade#922]
arkade get `inletsctl` fails in linux aarch64 and mingw

Installing inletsctl in GitBash (WSL) downloads the .exe file but is unable to copy it into the bin folder.

## Current Behaviour
Downloads `inletsctl.exe` but the copy command fails because it is looking for `inletsctl.exe.exe`

## Possible Solution
N/A

## Steps to Reproduce (for bugs)
Two ways to repro:

test-tool.sh
```
$ ./hack/test-tool.sh inletsctl
+ ./arkade get inletsctl --arch arm64 --os darwin --quiet
+ file .arkade/bin/inletsctl
/path/.arkade/bin/inletsctl: Mach-O 64-bit arm64 executable, flags:<|DYLDLINK|PIE>
+ rm /path/.arkade/bin/inletsctl
+ echo

+ ./arkade get inletsctl --arch x86_64 --os darwin --quiet
+ file /path/.arkade/bin/inletsctl
/path/.arkade/bin/inletsctl: Mach-O 64-bit x86_64 executable
+ rm /path/.arkade/bin/inletsctl
+ echo

+ ./arkade get inletsctl --arch x86_64 --os linux --quiet
+ file /path/.arkade/bin/inletsctl
/path/.arkade/bin/inletsctl: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=6M_Wagl7j2oAMAhNLegz/ElId8XoAfS4FFgo3RTUH/vdJIc5Q5GK5NVWWpn_rm/YKO4s5vPb3kc4txX14QC, stripped
+ rm /path/.arkade/bin/inletsctl
+ echo

+ ./arkade get inletsctl --arch aarch64 --os linux --quiet
Error: CreateFile /path/arkade-3336224941/inletsctl: The system cannot find the file specified.
+ ./arkade get inletsctl --arch x86_64 --os mingw --quiet
Error: CreateFile /path/arkade-2784826040/inletsctl.exe.exe: The system cannot find the file specified.
```

arkade get inletsctl
```
$ arkade get inletsctl
Downloading: inletsctl
2023/06/04 17:26:34 Looking up version for inletsctl
2023/06/04 17:26:34 Found: 0.8.21
Downloading: https://github.com/inlets/inletsctl/releases/download/0.8.21/inletsctl.exe.tgz
path/arkade-3701239706/inletsctl.exe.tgz written.
2023/06/04 17:26:35 Looking up version for inletsctl
2023/06/04 17:26:35 Found: 0.8.21
2023/06/04 17:26:36 Extracted: /path/arkade-3701239706/inletsctl.exe.exe
2023/06/04 17:26:36 Copying /path/arkade-3701239706/inletsctl.exe.exe to C:\Users\X/.arkade/bin/inletsctl.exe
Error: CreateFile /path/arkade-3701239706/inletsctl.exe.exe: The system cannot find the file specified.
```

## Your Environment

* What Kubernetes distribution are you using?

```
kubectl version
```

* Operating System and version (e.g. Linux, Windows, MacOS):

```
uname -a

cat /etc/os-release
```

* What arkade version is this?

```
arkade version
```

**Comment by alexellis** [comment]
Would you like to send a fix @joebowbeer ?

**Comment by joebowbeer** [comment]
@alexellis fixed in #924 - I also fixed aarch64, which was failing in test-tool.sh
